src/ui/component/Nav.html

```diff
@@ -21,17 +21,12 @@
             <li class="active"><a href="/profile">Profile</a></li>
         {:else}
             <li><a href="/profile">Profile</a></li>
-        {/if} 
-        {#if $req.originalUrl.includes("/skills")}
```

Comment:
Was this supposed to be deleted?

Reply:
Yes will be coming with rest of skills work